libs/templates/src/lib/trade/table/trading-spec-table.tsx

```diff
@@ -220,11 +220,61 @@ const TradingSpecTable = () => {
             />
           }
         >
-          <ActionSheetBottom
-            onChangeChips={handleChipsData}
-            onChangeBottomSheet={handleBottomSheet}
-            onChangeSelectedInfo={handleChipsInfo}
-          />
+          <div className=" flex flex-col p-general-md">
+            <div
+              className="flex flex-row items-center gap-gap-md px-general-md py-gap-md"
+              onClick={() => {
+                setSelectedFilter(mainInfo.data);
+                setSelectedInfo('mainInfo');
+                setShowBottomSheet(false);
+              }}
+            >
+              {selectedInfo === 'mainInfo' ? (
+                <StandaloneCircleDotFillIcon fill="black" iconSize="md" />
+              ) : (
+                <StandaloneCircleRegularIcon fill="black" iconSize="md" />
+              )}
+
+              <Text size="md" className="text-typography-default">
+                Main info
+              </Text>
+            </div>
+            <div
+              className="flex flex-row items-center gap-gap-md px-general-md py-gap-md"
+              onClick={() => {
+                setSelectedFilter(tradingCondition.data);
+                setSelectedInfo('tradingCondition');
+                setShowBottomSheet(false);
+              }}
+            >
+              {selectedInfo === 'tradingCondition' ? (
+                <StandaloneCircleDotFillIcon fill="black" iconSize="md" />
+              ) : (
+                <StandaloneCircleRegularIcon fill="black" iconSize="md" />
+              )}
+              <Text size="md" className="text-typography-default">
+                Trading conditions
+              </Text>
+            </div>
+            <div
+              className="flex flex-row items-center gap-gap-md px-general-md py-gap-md"
+              onClick={() => {
+                setSelectedFilter(additionalInfo.data);
+                setSelectedInfo('additionalInfo');
+                setShowBottomSheet(false);
+              }}
+            >
+              {selectedInfo === 'additionalInfo' ? (
+                <StandaloneCircleDotFillIcon fill="black" iconSize="md" />
+              ) : (
+                <StandaloneCircleRegularIcon fill="black" iconSize="md" />
+              )}
+
+              <Text size="md" className="text-typography-default">
```

Comment:
can you check if you remove `text-typography-default` will affect anything? By default this class will apply automatically

Reply:
yes can remove this. Thanks @jia-deriv